src/components/document/canvas.tsx

```diff
@@ -168,10 +168,12 @@ export class CanvasComponent extends BaseComponent<IProps, IState> {
 
   private handleDocumentUndo = () => {
     this.props.document?.undoLastAction();
+    return true;
```

Comment:
These `return true` statements seem to be required in order to make HotKeys' `dispatch` method call `preventDefault()` on the keyboard event.